images/prow-tests-go113/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20190716-41e2a9e-master
```

Comment:
If go1.13 is the only difference between the two images, you can based the image from the go1.12 stable version so we don't need to manage two sets of common dependencies.

Reply:
👍 agree.

Reply:
yeah was being lazy, will update this one

Reply:
Actually I realized that there should never be a case that the base `prow-test` and `prow-test-go113` images need to be pushed separately, I will figure out a way to make it happen

Reply:
Played with `Makefile` and `Dockerfile` for a couple of hours, trying to make it work with `/shared/Makefile.simple-image` but couldn't figure out a clean solution. @adrcunha , do you have any suggestion on making it work as `make push` pushes both `prow-tests` and `prow-tests-go113` images?

Reply:
My suggestion is that you don't fork prow-tests. Instead, use a pre-existing 1.12 image and pin to a specific version.

Reply:
then there will be a chance these 2 images diverge. If there is no better solution I will go with this one and add a comment there.
Thanks!

Reply:
The 1.12 version will be used only on old, stable branches, so it doesn't matter if it diverges from master, no?

Reply:
So far serving is the only repo requesting go1.13, this will be true if all repos opt in go1.13

Reply:
but I like the idea of not forking prow-tests, it would make the life much easier

Reply:
During this transition period, if it's easier just keeping two images, go ahead. Regarding the initial question, about `make push` doing two things, I would discourage that since the plan is that this is temporary.

Reply:
sg, updated to have 2 images for now with comment. Thanks for for suggestion.